addon/templates/components/polaris-pagination.hbs

```diff
@@ -1,39 +1,39 @@
-<button
-  data-test-prev-btn="true"
-  class="Polaris-Pagination__Button"
-  aria-label="Previous"
-  disabled={{isPreviousDisabled}}
-  onclick={{action onPrevious}}
-  onmouseup={{handleMouseUpByBlurring}}
+<nav
+  aria-label={{this.accessibilityLabel}}
+  class="Polaris-Pagination {{if @plain "Polaris-Pagination--plain"}}"
+  ...attributes
 >
-  {{polaris-icon source="arrow-left"}}
-</button>
+  <button
+    data-test-prev-btn="true"
+    class="Polaris-Pagination__Button"
+    aria-label="Previous"
+    disabled={{this.isPreviousDisabled}}
+    onclick={{fn (or @onPrevious this.onPrevious)}}
```

Comment:
same...let's use `on` modifier ([here](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/)'s a nice cheat-sheet you can reference)